TechAndCheck/zenodotus
# Send confirmation emails to applicant

This PR adds actual mail delivery of a "confirm your email!" message to the applicant confirmation process.

**Testing:**
- `bundle install`
- 🚨 Add the `MAILGUN_API_KEY` env var. You can find this in the 1Password vault as "Mailgun API (FCI/Development)". Use the `credential`, not the `username`.
- Start Sidekiq (used for async email delivery)
- `rails s`
- Fill out an application at http://localhost:3000/apply
- When prompted, check your inbox for an email
- Click the link inside the email
- ✅ You are confirmed! (Hopefully.) Verify on the Rails console: `applicant.confirmed?` should report true, and the `applicant[:confirmed_at]` attribute should be a timestamp instead of `nil`.

**Still TODO in future PRs:**
- Add tests (😢)
- Add design to HTML email
